app/src/main/java/com/poupa/vinylmusicplayer/provider/Discography.java

```diff
@@ -30,153 +36,166 @@
  * @author SC (soncaokim)
  */
 
-public class Discography extends SQLiteOpenHelper {
-    private static final String DATABASE_NAME = "discography.db";
-    private static final int VERSION = 1;
+public class Discography {
+    // TODO Genre editor.
+    // For some select song (ex Evanescence / Fallen album), changing genre to 'Heavy Metal' always turn it into 137 (instead of the text inserted).
+    // As if the change is redacted by JAudioTagger or the OS.
```

Comment:
I'm creating this PR to ask for feedback on this particular todo note.

I'm interested to know if this is something that can be reproduced on other setup (genre='Heavy Metal' is somehow saved/corrected as 137).

Reply:
This is solved with https://github.com/AdrienPoupa/VinylMusicPlayer/pull/292/commits/0c698d68609b12866b261793c0e2b49af834609f